buildAndReleaseTask/version.ts

```diff
@@ -4,7 +4,7 @@ import * as axios from "axios";
 import * as tl from "azure-pipelines-task-lib/task";
 
 export async function getLatestPulumiVersion(): Promise<string> {
-    const resp = await axios.default.get<string>("https://pulumi.io/latest-version", {
+    const resp = await axios.default.get<string>("https://www.pulumi.com/latest-version", {
```

Comment:
Ah, I wasn't aware that we had another dependency on https://pulumi.io/latest-version. Thanks for keeping an eye on this!

Note that we deliberately created an API endpoint for this (https://api.pulumi.com/api/cli/version) so that the CLI would not depend directly on https://pulumi.io/latest-version. Should we update this code to request https://api.pulumi.com/api/cli/version instead? (Note it returns JSON, so it's not a simple one line fix).

Though, it looks like our install scripts on https://get.pulumi.com make requests to https://pulumi.io/latest-version directly, though, in that case, presumably because it's easier to do that than parsing JSON.

Reply:
Ah I see. Does https://pulumi.io/latest-version end up actually calling https://api.pulumi.com/api/cli/version?

> Though, it looks like our install scripts on https://get.pulumi.com make requests to https://pulumi.io/latest-version directly

Yes, this is exactly why I ended up using that endpoint here.

Reply:
> Does https://pulumi.io/latest-version end up actually calling https://api.pulumi.com/api/cli/version?

No, it's the other way around. https://api.pulumi.com/api/cli/version calls https://pulumi.io/latest-version. We did this so the CLI wouldn't depend directly on the docs website.

Reply:
@justinvp I am going to leave this to call the `www` endpoint. If you are fine with that, can you please re-approve the PR?